LabDrivers/Tool.py

```diff
@@ -535,9 +535,14 @@ def change_interface(self, intf, **kwargs): # only really applicable to prologix
         else:
             logging.debug("Change interface called with %s, current interface is %s"%(intf, self.interface))
 
-
     def set_ren(self, level):
-        # this function is used as an alias for control_ren, which is built in to pyvisa controller, and programmed into Prologix
+        """
+        :param level: REN mode
+        :return: None
+        this function is used as an alias for control_ren, which is built in to pyvisa controller, and programmed into
+         Prologix, and implemented in Serial
+
+        """
```

Comment:
I would direct you to https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings for the proper docstring formatting :), apart from that you can merge :)